src/struct_info.json

```diff
@@ -33,7 +33,7 @@
         }
     },
     {
-        "file": "libc/sys/stat.h",
+        "file": "sys/stat.h",
```

Comment:
what do these changes do?

Reply:
This is because the include order changed a little:

It went from:

```
#include "..." search starts here:
#include <...> search starts here:
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/SDL
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/compat
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/libc
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/libc/musl/arch/emscripten
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/local/include
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/SSE
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/neon
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/compiler-rt/include
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/libunwind/include
 /usr/local/google/home/sbc/dev/wasm/llvm-build/lib/clang/12.0.0/include
End of search list.
```

```
#include "..." search starts here:
#include <...> search starts here:
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/SDL
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/compat
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/libc
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/libc/musl/arch/emscripten
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/local/include
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/SSE
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include/neon
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/compiler-rt/include
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/libunwind/include
 /usr/local/google/home/sbc/dev/wasm/llvm-build/lib/clang/12.0.0/include
 /usr/local/google/home/sbc/dev/wasm/emscripten/system/include
End of search list.
```

Note that `/usr/local/google/home/sbc/dev/wasm/emscripten/system/include` went to the bottom.    The reason it went to the bottom is that is now part of the default sysroot include path, and not explictly added by emscripten anymore (the nunber of explictly needed include paths went down by one).    But since all explicitly include paths come first, this one now comes last.

That means that if you include ou include `<compat/sys/stat.h>` explicitly (which nobody should be doing) it doesn't work because it wants to "#include_next <sys/stat.h>" which can never work if that file was originally found via the final element of the include path.

I will add a comment about this..